src/pages/Login/Login.jsx

```diff
@@ -0,0 +1,80 @@
+import { useRef, useState } from 'react';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import Input from '../../components/Input';
+import './Login.css';
+
+export default function Login() {
+  const usernameInput = useRef();
+  const passwordInput = useRef();
+  const [validForm, setValidForm] = useState(false);
+
+  function login() {
+    const loginData = {
+      username: usernameInput.current.value,
+      password: passwordInput.current.value,
+    };
+
+    console.log(loginData);
+  }
+
+  function checkValidation() {
+    const passwordVal = passwordInput.current.value;
+    const [username, companyName] = usernameInput.current.value.split('@');
+
+    setValidForm(
+      isValidUsername(username) &&
+        isValidUsername(companyName) &&
```

Comment:
Nega  isvalidusername ga companyname bervosan?

Keyin vapsheta   
azimjon@bandla
San shuni ozini Yani loggini ni tekshir boldi
azimjon ni alohida  bandla ni alohida tekshirma

Example
azimjon@bandla -> valid
azimjonbandla -> not valid 
azimjonbandla@bandla -> valid

Reply:
Example code

Reply:
> Nega isvalidusername ga companyname bervosan?

Company name tekshirish keremasmi?

Reply:
Bitta qb tekshir dvomau , alohida qimasdan